clouddriver-openstack/src/main/groovy/com/netflix/spinnaker/clouddriver/openstack/deploy/description/servergroup/ServerGroupParameters.groovy

```diff
@@ -105,15 +115,24 @@ class ServerGroupParameters {
    */
   static List<String> unescapePythonUnicodeJsonList(String string) {
     string?.split(",")?.collect { s ->
-      s.replace("u'", "")
-        .replace("'","")
-        .replace("[","")
-        .replace("]","")
-        .replaceAll("([ ][ ]*)","")
+      s.replace("u'", "").replace("'","").replace("[","").replace("]","").replaceAll("([ ][ ]*)","")
```

Comment:
Is this still needed if we are only supporting Mitaka? Or is that bug still in Mitaka?